dev-requirements.txt

```diff
@@ -3,6 +3,7 @@ miniboa
 parameterized
 pdfminer
 pyftpdlib
+pyopenssl
```

Comment:
Huh, I see we're already using pyopenssl (the vendored copy from requests):

```
linkcheck/httputil.py:    from requests.packages.urllib3.contrib.pyopenssl import get_subj_alt_name
```

I wonder if we should change that to `from pyopenssl import get_subj_alt_name`, or if we should continue using requests.packages.urllib3.contrib.pyopenssl for new pyopenssl api calls.

Incidentally, where are they?  I don't see them in this diff.

Reply:
> Basically ignore me, I've just shown how unqualified I'm to review pull requests :)

I think that's amazingly inaccurate and probably the only relevant inaccuracy in your reviews. :p You rock, don't say stuff like that. ;)